package.json

```diff
@@ -13,7 +13,7 @@
     "eslint": "eslint .",
     "flow:check": "flow check",
     "test": "jest --config=jest.json",
-    "test-ci": "cd storybooknative; yarn install; cd ..; yarn flow:check && yarn eslint && yarn test --ci",
+    "test-ci": "cd storybooknative; yarn install; cd ..; yarn flow:check && yarn eslint && yarn test --ci; cd storybooknative; yarn test",
```

Comment:
I added this; this is hacky for now and because `jest` needs `babel.config.js` when the rest is happy with `.babelrc`